test/specs/rules/validate-indentation.js

```diff
@@ -156,6 +156,23 @@ describe('rules/validate-indentation', function() {
         ]);
     });
 
+    it('should not report errors for using tabs when validateIndentation is a number', function() {
+        checker.configure({ validateIndentation: 4 });
```

Comment:
Checked the code from https://github.com/jscs-dev/node-jscs/issues/1907 it still shows same errors